components-styles.css

```diff
@@ -0,0 +1,578 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+/* AVATAR */
+
+.border-img{
```

Comment:
it would be great if you put width height in a class so you don't have to rewrite it again and you can use that class with other classes also try to use in rem  instead of px will be best practice